### DISCUSSION TITLE
Open database of intentional communities (and coliving projects?) for research purposes

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#144] [Research]
idea: create an open database of intentional communities (and associated research project) especially one with **historical** data.

We're very interested in co-living and intentional communities -- and what makes them work or not work -- so a database of them would be very useful. especially one focused on historical data.

See WIP here https://github.com/life-itself/intentional-communities

## Brainstorm

* [ ] Identify research questions we are interested in e.g.
  * [ ] What is the actual success/failure rate of intentional communities (and their time in existence)? People quote rough numbers like 90% of intentional communities fail - cf https://lifeitself.us/2021/05/14/notes-on-creating-a-life-together-by-diana-leafe-christian/
  * [ ] What factors influence the success and failure of communities? (e.g. religious / non-religious - cf sosis paper)
* [ ] Start collecting a database from various sources
  * [ ] identify sources e.g. Oved, https://ic.org, others
  * [ ] scrape and consolidate data
  * [ ] start an online browseable database/wiki

**Comment by Liyubov** [comment]
<comment>
 
Documenting some progress here regarding the issue #144:

- adding the link to the small data-viz of open data of communities which collected and mentioned [here](https://github.com/life-itself/intentional-communities).

I am documenting some visualisations and research around it on it on my local [branch](https://github.com/Liyubov/community/tree/master/research), trying to do survivability analysis for communities to estimate which types of communities would survive longer and based on which criteria. plan is to merge later with the main branch of life-itself research;

- there is a more recent database of communities on [ic.org](http://ic.org/), which would be good to analyse: e.g. using open API. 

- another resource list available at [wikipedia](https://en.wikipedia.org/wiki/List_of_intentional_communities).

Any suggestions are welcome.

</comment>

**Reply by Liyubov** [reply]
**status update**: 

- waiting for the reply from ic.org on sample list of communities data
- data-viz of open data of communities from the book and collected [here](https://github.com/life-itself/intentional-communities)

**Reply by rufuspollock** [reply]
Great work @Liyubov and i think this is a really great thing to move forward. Whilst waiting let's start documenting the sources we could find. I also think a zotero group could be good for this too!

**Reply by rufuspollock** [reply]
@Liyubov any updates here?

Shall we start a zotero group for this where we can track sources?

**Reply by baouroux** [reply]
A couple of recent references I came across here on the topic....
- Visions for ecotopia living in fiction: "An East-West Psychology Dissertation Defense by Ryan “Indigo” Warman (at the Californian Institute For Integral Studies) CIIS: Visions of the Great Turning: Utopianism and Conscious Evolution in New Age Science Fiction" 
- A Canadian TV channel on intentional communities and neighborhoods: https://peakmoment.tv/about/ (a filming associate of mine has filmed one of the key contributors.

**Reply by rufuspollock** [reply]
@baouroux very useful. BTW would you like to introduce yourself in the introductions channel https://github.com/life-itself/community/discussions/categories/introductions - just create a new discussion and say hi and share a bit about you (if you want!)

**Reply by rufuspollock** [reply]
@Liyubov any updates here? Did we get db from ic.org?

**Reply by rufuspollock** [reply]
@Liyubov any update on access  to ic.org database?

**Reply by rufuspollock** [reply]
UPDATE: i believe update ~1y ago from @Liyubov was that IC had written back saying they couldn't share openly (even in anonymised form) but perhaps for research purposes they could (and i think at that point the thread dropped)

**Comment by BoazBF** [comment]
Hey @rufuspollock @Liyubov & @baouroux 
I'm delighted to join you here in brainstorming ways to collect data about best practices for community-building. This is a topic I'm particularly interested in, and am endeavouring maybe 

- exploring ethnographic data collection in some of the main communities in Europe (Siebenlinden, Damanhur, Tamera, Finhorn, etc).
- Digging into major communities in EUrope which has crumbled, identifying key issues

Will keep you posted on this!

**Reply by rufuspollock** [reply]
@BoazBF this is great! 🎉 I think we could set up a open planning call in the next few weeks to plan some next steps and how people can contribute in various ways.

**Comment by almereyda** [comment]
Just to let you know, I'm subscribed to this issue and ready to contribute experiences from the @transformap effort, once they become useful.

**Reply by rufuspollock** [reply]
@almereyda that's really appreciated! Thank-you.

**Comment by rufuspollock** [comment]
Flagging that we now have a semi-duplicate new discussion here https://github.com/orgs/life-itself/discussions/391

**Reply by nathenf** [reply]
Yes good point. Shall we merge them? I feel like this speaks more to one job stories for having a database which is to carry out research. Definitely worth keeping in mind as we scrap which parameters we want to research, such as religious affiliations.

**Comment by msantolini** [comment]
Hello there!

I somehow got in the depths of my emails and found this thread, which picked my interest :).

I downloaded the data from https://github.com/life-itself/intentional-communities and looked at it quickly. Little digest below 👇 

![image](https://github.com/life-itself/community/assets/956597/b3e84594-7e37-45b1-a59b-6c904c7d1a14)

The database stops in 1878, and documents communities until "present". Looking at the cumulative number, we see a burst of activity in the 1850s.

![image](https://github.com/life-itself/community/assets/956597/e4852ac3-df6a-451d-b3c0-d46887de4f09)

Now looking at the "duration active" variable (difference of end and start date), we see a lot of very long-term communities were founded in late 1790s (blue dots above). Wondering about why it is so, I split the data by type, i.e.

> R=Religious
> O=Owenites
> S= Socialists
> M=Miscellaneous
> SH=Shakers
> F=Fourierists
> AN=Anarchists


![image](https://github.com/life-itself/community/assets/956597/308a5270-914f-47a8-9257-39aadd598789)
![image](https://github.com/life-itself/community/assets/956597/e53f0b63-d349-479e-acac-2804ed9fff68)


The very **long-standing communities are overwhelmingly religious, most particuarly the [Shakers](https://en.wikipedia.org/wiki/Shakers)**. This is true both at the average level (barplot), as well as the individual level (boxplot).

And to finish, with the question of "90% of communities failing", obviously the data collection has a bias of collecting points we remember, therefore those who didn't fail... But just as an exercise, in the database, **72% of communities are less than one generation (20 years)**. Maybe the rule could be more of a revisited 80-20, i.e. 80% of communities lasts less than 20 years 😊 .

**Comment by baouroux** [comment]
Fascinating Marc....

I might have posted this before, but on a similar theme, I remember this
talk (audio link below) by Rosabeth Moss Kanter speaking of her research
into the longevity of intentional communities, and the fact that it was
very rarely financial issues that lead to their demise, but more the
ideals, purpose and identity that fragmented (something the more explicit
faith/spiritual tradition communities I guess manage to sustain):

- https://archive.org/details/RosabethKanterG8

This comes out of a series of stunning talks as part of the Lindisfarne
community conferences, going way back to the 70s, but just as enlightening
as ever I feel... With William Irwin Thompson's brilliant cultural
anthropology presentations. The whole archive of audios from their
conferences can be found here!
https://centerforneweconomics.org/envision/legacy/lindisfarne-tapes/

**Reply by msantolini** [reply]
Thanks as always for the enlightening contextualisation! 🙏

**Comment by baouroux** [comment]
Just an addition to the list of communities....I have a fellow portrait filmmaker in Canada who was commissioned to film the portrait of the founder of this community:
-  https://ourecovillage.org/ in British Columbia, 
and their "re-villaging process".

Here is the film:  Re-becoming Villagers:  https://vimeo.com/493144192?share=copy Very inspiring to have the personal stories....
